File: cpp/35b_1873A_ShortSort.cpp
#include<bits/stdc++.h>

int main(){
    int t, cnt;
    std::vector<std::string> ans;
    std::unordered_map<char, int> hmap;
    hmap['a'] = 0;
    hmap['b'] = 1;
    hmap['c'] = 2;
    std::string str;
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>str;
        cnt = 0;
        for(int j = 0; j < 3; j++){
            if(hmap[str[j]] != j){
                cnt++;
            }
        }
        if (cnt <= 2){
            ans.push_back("YES");
        }
        else{
            ans.push_back("NO");
        }
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/35b_1873A_ShortSort.py
def main():
    ans = []
    hmap = {'a' : 0,
            'b' : 1,
            'c' : 2}
    t = int(input())
    for _ in range(t):
        s = input().strip()
        cnt = 0
        for i in range(3):
            if hmap[s[i]] != i:
                cnt += 1
        if cnt <= 2:
            ans.append("YES")
        else:
            ans.append("NO")
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()